skimage/measure/_label.py

```diff
@@ -29,7 +30,8 @@ def _label_bool(image, background=None, return_num=False, connectivity=None):
         return result[0]
 
 
-def label(input, background=None, return_num=False, connectivity=None):
+@deprecate_kwarg({"input": "image"}, removed_version="1.0")
+def label(image, background=None, return_num=False, connectivity=None):
```

Comment:
I think this is actually a `label_image`... =)

Reply:
Done, Thank you @jni :wink: